content/kubernetes/re-clusters/redis-on-flash.md

```diff
@@ -0,0 +1,83 @@
+---
+Title: Use Redis on Flash on Kubernetes
+linkTitle: Redis on Flash
+description: Deploy and configure a Redis cluster or database on Kubernetes that supports Redis on Flash.
+weight: 16
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/redis-on-flash.md,
+    /kubernetes/re-clusters/redis-on-flash/,
+]  
+---
+
+## Prerequisites
+
+Redis Enterprise Software for Kubernetes supports using NVMe (nonvolatile memory express) SSDs (solid state drives) for flash storage with the Redis on Flash feature.
```

Comment:
```suggestion
Redis Enterprise Software for Kubernetes supports Redis on Flash, which extends memory capacity on SSDs or flash (NVMe).
```

Reply:
> 

I'll have to double check, but my understanding is that flash memory needs to be SSDs and our operator requires those SSDs be of the NVMe variety.

Reply:
I don't think that we should state in the prerequisites section that NVMe is a strong requirement or the only supported flash drive type.
The only requirements are:
1. SSDs must be locally attached to the worker nodes (not network attached, network attached storage is not supported for RoF case).
2. SSDs must be formatted and mounted on the worker nodes.
3. Provisioned as local persistent volumes.
4. Assigned a unique `StorageClass`.

Maybe the right thing to do is to state that using high-performing NVMe is our recommendation to achieve best performance.